### ISSUE TITLE
[x86_64] "self-upgrade" pulls aarch64

### ISSUE THREAD

**Issue by cinnamonmatexfce** [zyrouge/pho#7]
```
$ file pho && pho self-upgrade && file pho
pho: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=sWyTbAT3EXzqqlvK5vAc/7OUsz6o-qD4ktlvEnRqk/Xnnu_BZduD4XZhZ2EDRC/MtRsvWIM0wQGjBOjiDnh, stripped
Updating to version v0.1.3...
√ Updated to version v0.1.3 successfully!
pho: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, Go BuildID=mEAyg7OimPRbsBRb3ntx/IqDhNCpvYTT4VELaBJiH/PE88KB6262qhWrKifAHT/u_gLJLMeHpl8dkE8S5XN, stripped
```

```
$ pho self-upgrade || file pho 
bash: .local/bin/pho: cannot execute binary file: Exec format error
pho: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, Go BuildID=mEAyg7OimPRbsBRb3ntx/IqDhNCpvYTT4VELaBJiH/PE88KB6262qhWrKifAHT/u_gLJLMeHpl8dkE8S5XN, stripped
```

**Comment by zyrouge** [comment]
> ```
> $ file pho && pho self-upgrade && file pho
> pho: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=sWyTbAT3EXzqqlvK5vAc/7OUsz6o-qD4ktlvEnRqk/Xnnu_BZduD4XZhZ2EDRC/MtRsvWIM0wQGjBOjiDnh, stripped
> Updating to version v0.1.3...
> √ Updated to version v0.1.3 successfully!
> pho: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, Go BuildID=mEAyg7OimPRbsBRb3ntx/IqDhNCpvYTT4VELaBJiH/PE88KB6262qhWrKifAHT/u_gLJLMeHpl8dkE8S5XN, stripped
> ```
> 
> ```
> $ pho self-upgrade || file pho 
> bash: .local/bin/pho: cannot execute binary file: Exec format error
> pho: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, Go BuildID=mEAyg7OimPRbsBRb3ntx/IqDhNCpvYTT4VELaBJiH/PE88KB6262qhWrKifAHT/u_gLJLMeHpl8dkE8S5XN, stripped
> ```

That's weird, what does `uname -a` output?

**Comment by zyrouge** [comment]
Fixed as of v0.1.4.
